airbyte-integrations/connectors/destination-iceberg/build.gradle

```diff
@@ -14,21 +14,30 @@ dependencies {
     implementation project(':airbyte-integrations:bases:base-java')
     implementation files(project(':airbyte-integrations:bases:base-java').airbyteDocker.outputs)
 
-    implementation('org.apache.spark:spark-sql_2.13:3.3.0') {
+    implementation('org.apache.spark:spark-sql_2.13:3.3.1') {
         exclude(group: 'org.apache.hadoop', module: 'hadoop-common')
     }
-    implementation('org.apache.spark:spark-hive_2.13:3.3.0') {
+    implementation('org.apache.spark:spark-hive_2.13:3.3.1') {
         exclude(group: 'org.apache.hadoop', module: 'hadoop-common')
     }
-    implementation 'org.apache.iceberg:iceberg-spark-runtime-3.3_2.13:1.0.0'
-    implementation "software.amazon.awssdk:bundle:2.17.131"
-    implementation "software.amazon.awssdk:url-connection-client:2.17.131"
+    implementation 'org.apache.iceberg:iceberg-spark-runtime-3.3_2.13:1.1.0'
+
+    // force awssdk version required by Iceberg
+    implementation "software.amazon.awssdk:utils:2.17.257"
+    implementation "software.amazon.awssdk:url-connection-client:2.17.257"
+    implementation "software.amazon.awssdk:s3:2.17.257"
+    implementation "software.amazon.awssdk:glue:2.17.257"
+    implementation "software.amazon.awssdk:dynamodb:2.17.257"
+    implementation "software.amazon.awssdk:kms:2.17.257"
+    implementation "software.amazon.awssdk:sts:2.17.257"
+    implementation "software.amazon.awssdk:sdk-core:2.17.257"
+    implementation "software.amazon.awssdk:aws-core:2.17.257"
```

Comment:
these must all match the AWS SDK version that Iceberg 1.1.0 uses: https://github.com/apache/iceberg/blob/apache-iceberg-1.1.0/versions.props#L24